packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md

```diff
@@ -0,0 +1,116 @@
+
+You use the Org AS to perform SSO with Okta or to get an access token for the Okta APIs. You can't customize this authorization server with regards to audience, claims, policies, or scopes. Additionally, the resulting access token's issuer is `https://${yourOktaOrg}`, which indicates that only Okta can consume or validate it. The access token can't be used or validated by your own applications.
+
+#### Org Authorization Server discovery endpoints
+
+The following discovery endpoints return OIDC or OAuth 2.0 metadata related to your Org AS. Clients can use this information to programmatically configure their interactions with Okta. Custom scopes and custom claims aren't returned.
+
+**OpenID:** `https://${yourOktaOrg}/.well-known/openid-configuration`
+
+**OAuth:** `https://${yourOktaOrg}/.well-known/oauth-authorization-server`
+
+### Custom Authorization Server
+
+You use a Custom Authorization Server to create and apply authorization policies to secure your APIs. An access token that is minted by a Custom Authorization Server is consumed by your APIs.
+
+Okta allows you to create multiple Custom Authorization Servers within a single Okta org that you can use to protect your own resource servers. Within each authorization server, you can define your own custom OAuth 2.0 scopes, claims, and access policies to support authorization for your APIs.
+
+#### Default Custom Authorization Server
+
+Okta provides a pre-configured Custom Authorization Server called `default`. It includes a basic access policy and a rule to quickly get you started. For simple use cases, this out-of-the-box Custom Authorization Server is usually all that you need.
+
+To use the `default` Custom Authorization Server, use `default` as the authorization server ID:
+
+`https://${yourOktaDomian}/api/v1/authorizationServers/default`
+
+For Custom Authorization Servers that you create yourself, the `${authServerId}` is a random ID such as `aus9o8wzkhckw9TLa0h7z`.
+
+`https://${yourOktaDomain}/api/v1/authorizationServers/${authServerId}`
+
+#### Custom Authorization Server discovery endpoints
+
+The following endpoints return OIDC or OAuth 2.0 metadata related to a Custom Authorization Server. Clients can use this information to programmatically configure their interactions with Okta. Custom scopes and custom claims aren't returned.
+
+The OpenID and OAuth discovery endpoints for a Custom Authorization Server are:
+
+**OpenID:** `https://${yourOktaDomain}/oauth2/<server id>/.well-known/openid-configuration`
+
+**OAuth:** `https://${yourOktaDomain}/oauth2/<server id>/.well-known/oauth-authorization-server`
+
+The OpenID and OAuth discovery endpoints for the default Custom Authorization Server are:
+
+**OpenID:** `https://${yourOktaDomain}/oauth2/default/.well-known/openid-configuration`
+
+**OAuth:** `https://${yourOktaDomain}/oauth2/default/.well-known/oauth-authorization-server`
+
+## Which authorization server should you use
```

Comment:
Somewhere, we should mention that using customAS will impact your billing. OrgAS comes with your Org and CustomAS costs extra. I wouldn't focus on this, but having a link to billing or mention that it would impact the cost of the org, so that it doesn't come as a surprise for customers.